### ISSUE TITLE
I just can't figure out how to put image datasets and mask datasets

### ISSUE THREAD

**Issue by biexiangduo** [lyndonzheng/Pluralistic-Inpainting#53]
sorry, I just can not figure out where to store image and where to store irregular mask. And I see three pictures in every kind of datasets.So i directly start train without any movement.After half day and a night, I stop training beacuse i can not understand the generated data.

At last, I donwloaded your pretrained model and put it to checkpoint, after gui launch successly, I draw mask and fill, result is strange ,like this: 
![Screenshot from 2020-08-11 17-31-06](https://user-images.githubusercontent.com/27905256/89881763-82bcb900-dbf8-11ea-9ccd-781581b39f4b.png)

Is it normal?

**Comment by lyndonzheng** [comment]
The result is strange. You may load the wrong model. This is a duplicate of https://github.com/lyndonzheng/Pluralistic-Inpainting/issues/9, FYI.

**Comment by biexiangduo** [comment]
> The result is strange. You may load the wrong model. This is a duplicate of #9, FYI.

Thank for your answer sincerely! Could I understand the strange output caused by my unsufficient training?

**Comment by lyndonzheng** [comment]
If this visual result comes from our pre-trained model, you must run and test the model in the wrong way. The results should be similar to our [online demo](http://www.chuanxiaz.com/project/pluralistic/).
